controllers/ProfileController.js

```diff
@@ -69,6 +69,35 @@ class ProfileController {
       user: userData
     });
   }
+
+  /**
+   * @author Manzi
+   * @param {*} req
+   * @param {*} res
+   * @returns {*} Users object
+   */
+  static async getProfiles(req, res) {
+    const profiles = await User.findAll({attributes: ['firstName', 'lastName', 'image', 'bio', 'cover']});
+    return res.status(200).json({ status: 200, user: profiles });
```

Comment:
@NkFab As `/api/v1/profiles` is the ressource that we are trying to fetch, the response body should be `{ status: 200, profiles }` instead of  `{ status: 200, user }`. Remember that when fetching a list of something, the attribute has to be in plural. For instance if the resource was `/api/v2/users`, we were going to have this reponse body `{ status: 200, users }`